components/onboarding/internal/adapters/http/in/account.go

```
@@ -123,6 +123,7 @@ func (handler *AccountHandler) GetAllAccounts(c *fiber.Ctx) error {
 	ledgerID := c.Locals("ledger_id").(uuid.UUID)
 
 	var portfolioID *uuid.UUID
+	var segmentID *uuid.UUID
```

golangci-lint (2.11.3)

[error] missing whitespace above this line (never cuddle decl)

(wsl_v5)